test-requirements.txt

```diff
@@ -4,3 +4,4 @@ pytest-cov==2.11.1
 isort==5.8.0
 autopep8==1.5.7
 flake8==3.9.1
+codecov==2.1.11
```

Comment:
I don't think we're using codecov in the unit tests

Reply:
Yep, we are currently not using codecov, However, once we move this to the AYX org, I will add start using codecov to generate a report.